frontend/src/views/groupmember/AddGroupMember.vue

```diff
@@ -0,0 +1,313 @@
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.homeAddress.addressLine3" id="addressLine3" label="Line 3 (Optional)" type="text" v-model="homeAddress.addressLine3" />
+        </AppCol>
+      </AppRow>
+       <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.homeAddress.addressLine4" id="addressLine4" label="Line 4 (Optional)" type="text" v-model="homeAddress.addressLine4" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput :e-model="v$.homeAddress.postalCode" id="postalCode" label="Postal Code" type="text" v-model.trim="homeAddress.postalCode" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.mailingAddress.addressLine1" id="mailingAddress1" label="Mailing Address (if different from home address)" v-model="mailingAddress.addressLine1" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.mailingAddress.addressLine2" id="mailingAddress2" label="Line 2 (Optional)" v-model="mailingAddress.addressLine2" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.mailingAddress.addressLine3" id="mailingAddress3" label="Line 3 (Optional)" v-model="mailingAddress.addressLine3" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col7">
+          <AppInput :e-model="v$.mailingAddress.addressLine4" id="mailingAddress4" label="Line 4 (Optional)" v-model="mailingAddress.addressLine4" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col4">
+          <AppInput :e-model="v$.mailingAddress.postalCode" id="mailingPostalCode" label="Postal Code" type="text" v-model.trim="mailingAddress.postalCode" />
+        </AppCol>
+      </AppRow>
+      <div>
+        <AppRow>
+          <AppCol class="col4"><h2>Dependent(s) (Optional)</h2>
+          </AppCol>
+          <AppCol class="col4">
+          </AppCol> 
+        </AppRow>       
+        <AppRow>
+          <AppCol class="col3"><b>Relationship</b>
+          </AppCol>
+          <AppCol class="col3"><b>PHN</b>
+          </AppCol> 
+        </AppRow> 
+        <AppRow>
+          <AppCol class="col3"><b> Spouse </b>
+          </AppCol>
+          <AppCol class="col3">
+            <AppInput :e-model="v$.spousePhn" id="spousePhn" type="text" v-model.trim="spousePhn" />
+          </AppCol> 
+        </AppRow>
+        <AppRow>
+          <AppCol class="col4">
+          </AppCol> 
+        </AppRow>      
+      </div> 
+      <AddListDependent :dependents="dependents" @add-dependent="addDependent" @remove-dependent="removeDependent"/>
+      <AppRow>
+        <AppButton :submitting="submitting" mode="primary" type="submit">Submit</AppButton>
+        <AppButton @click="resetForm()" mode="secondary" type="button">Clear</AppButton>
```

Comment:
Clear button doesn't clear content in the Dependent PHN field but not sure if this is significant as this data is not submitted directly.